entity_matching/template_3dnodes_assets.py

```diff
@@ -0,0 +1,248 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.3.2
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %load_ext autoreload
+# %autoreload 2
+
+# +
+import os
+import json
+
+import numpy as np
+import pandas as pd
+from cognite.client import CogniteClient
+
+from cognite.client.data_classes.three_d import ThreeDAssetMapping
+
+from cognite.datastudio.entity_matcher import EntityMatcher
+
+# +
+# we provide functions to simplify the logics of the notebook
+from utils import chunk_create_rules_df, chunk_predict, get_matches_with_rules
+
+# we provide functions to install 3d-nodes, assets and asset-mappings
+from data_load_cdf import load_assets, load_threednodes, filter_df_threednodes, load_asset_mappings
+# -
+
+# # Initialize
+
+project = "akerbp"
```

Comment:
This seems way too specific for `akerbp`. I think we need a simpler example.

Reply:
This is the same for other projects I believe. What was to specific besides the name of the project?

Reply:
Changed the project and key name.